pipenv created Python virtual environments show in quotes

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

When a Python virtual environment that has been created with pipenv is activated, the `.Venv` name shows in quotes. This does not occur when using `uv` or standard library module 'venv`. I have not tried other Python virtual environment managers. This only recently started occurring though I didn't note exactly when. Sometime within the past week or two possibly.

## Steps to replicate:

```
# In a new blank directory...
export PIPENV_VENV_IN_PROJECT=1
pipenv install
source .venv/bin/activate
```

when using the following theme template:

`"template": "  {{ if .Error }}{{ .Error }}{{ else }}{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Full }}{{ end }}",`

shows the following:

<img width="308" height="114" alt="Image" src="https://github.com/user-attachments/assets/ea17db1a-544c-4b48-9ccd-530a05ebfa25" />

Entirely possible this is a pipenv bug as well...

### Theme

{
    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
    "blocks": [
        {
            "alignment": "left",
            "segments": [
                {
                    "background": "#21c7a8",
                    "foreground": "#011627",
                    "leading_diamond": "\u256d\u2500\ue0b6",
                    "options": {
                        "windows": "\ue62a"
                    },
                    "style": "diamond",
                    "template": " {{.Icon}} ",
                    "type": "os"
                },
                {
                    "background": "#ef5350",
                    "foreground": "#ffeb95",
                    "powerline_symbol": "\ue0b0",
                    "style": "powerline",
                    "template": " \uf292 ",
                    "type": "root",
                    "min_width": 170
                },
                {
                    "background": "#82AAFF",
                    "foreground": "#011627",
                    "powerline_symbol": "\ue0b0",
                    "options": {
                        "folder_icon": "\uf07c ",
                        "folder_separator_icon": "<#011627>\ue0b1</> ",
                        "home_icon": " \ueb06 ",
                        "style": "agnoster_short"
                    },
                    "max_depth": 3,
                    "style": "powerline",
                    "template": "{{ path .Path .Location }} ",
                    "type": "path",
                    "min_width": 160
                },
                {
                    "background": "#82AAFF",
                    "foreground": "#011627",
                    "powerline_symbol": "\ue0b0",
                    "options": {
                        "folder_icon": "\uf07c ",
                        "folder_separator_icon": "<#011627>\ue0b1</> ",
                        "home_icon": " \ueb06 ",
                        "style": "folder"
                    },
                    "style": "powerline",
                    "template": "{{ path .Path .Location }} ",
                    "type": "path",
                    "max_width": 159
                },
                {
                    "background": "#addb67",
                    "background_templates": [
                        "{{ if or (.Working.Changed) (.Staging.Changed) }}#e4cf6a{{ end }}",
                        "{{ if and (gt .Ahead 0) (gt .Behind 0) }}#f78c6c{{ end }}",
                        "{{ if gt .Ahead 0 }}#C792EA{{ end }}",
                        "{{ if gt .Behind 0 }}#c792ea{{ end }}"
                    ],
                    "foreground": "#011627",
                    "powerline_symbol": "\ue0b0",
                    "options": {
                        "branch_icon": "\ue725 ",
                        "fetch_status": true,
                        "fetch_upstream_icon": true,
                        "mapped_branches": {
                            "feat/*": "🚀 ",
                            "bug/*": "🐛 "
                        }
                    },
                    "style": "powerline",
                    "template": " {{ .UpstreamIcon }} {{ .HEAD }}{{if .BranchStatus }} {{ .BranchStatus }}{{ end }}{{ if .Working.Changed }} \uf044 {{ .Working.String }}{{ end }}{{ if and (.Working.Changed) (.Staging.Changed) }} |{{ end }}{{ if .Staging.Changed }} \uf046 {{ .Staging.String }}{{ end }}{{ if gt .StashCount 0 }} \ueb4b {{ .StashCount }}{{ end }} ",
                    "type": "git"
                },
                {
                    "background": "#575656",
                    "foreground": "#d6deeb",
                    "options": {
                        "style": "roundrock",
                        "threshold": 0
                    },
                    "style": "diamond",
                    "template": " {{ .FormattedMs }}",
                    "trailing_diamond": "\ue0b4",
                    "type": "executiontime"
                }
            ],
            "type": "prompt"
        },
        {
            "alignment": "right",
            "overflow": "break",
            "segments": [
                {
                    "background": "#565656",
                    "foreground": "#faa029",
                    "leading_diamond": "\ue0b2",
                    "style": "diamond",
                    "template": "\ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }}",
                    "type": "aws",
                    "min_width": 150
                },
                {
                    "background": "#316ce4",
                    "foreground": "#ffffff",
                    "leading_diamond": "\ue0b2",
                    "options": {
                        "parse_kubeconfig": true
                    },
                    "style": "diamond",
                    "template": "\ue81d {{.Context}}{{if .Namespace}} :: {{.Namespace}}{{end}}",
                    "type": "kubectl",
                    "min_width": 160
                },
                {
                    "background": "#303030",
                    "foreground": "#3C873A",
                    "leading_diamond": "\ue0b2",
                    "options": {
                        "fetch_package_manager": true,
                        "npm_icon": "<#cc3a3a>\ue71e</> ",
                        "yarn_icon": "<#348cba>\ue6a7</> "
                    },
                    "style": "diamond",
                    "template": "\ue718 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }}",
                    "type": "node",
                    "min_width": 150
                },
                {
                    "background": "#306998",
                    "foreground": "#FFE873",
                    "leading_diamond": "\ue0b2",
                    "style": "diamond",
                    "template": "  {{ if .Error }}{{ .Error }}{{ else }}{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Full }}{{ end }}",
                    "type": "python",
                    "min_width": 150
                },
                {
                    "background": "#ffffff",
                    "foreground": "#000000",
                    "leading_diamond": "\ue0b2",
                    "style": "diamond",
                    "template": "\ue7a8 {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} ",
                    "type": "rust",
                    "min_width": 150
                },
                {
                    "background": "#234d70",
                    "foreground": "#d6deeb",
                    "leading_diamond": "\ue0b2",
                    "options": {
                        "time_format": "15:04:05"
                    },
                    "style": "diamond",
                    "template": "\ue641 {{ .CurrentDate | date .Format }} ",
                    "type": "time"
                },
                {
                    "background": "#234d70",
                    "foreground": "#d6deeb",
                    "leading_diamond": "\ue0b2",
                    "style": "diamond",
                    "template": "\uef5f {{ dateInZone \"15:04Z\" .CurrentDate \"UTC\" }}",
                    "trailing_diamond": "\ue0b4",
                    "type": "time"
                }
            ],
            "type": "prompt"
        },
        {
            "alignment": "left",
            "newline": true,
            "segments": [
                {
                    "foreground": "#21c7a8",
                    "style": "plain",
                    "template": "\u2570\u2500",
                    "type": "text"
                },
                {
                    "foreground": "#22da6e",
                    "foreground_templates": [
                        "{{ if gt .Code 0 }}#ef5350{{ end }}"
                    ],
                    "options": {
                        "always_enabled": true
                    },
                    "style": "plain",
                    "template": "\ue285\ue285",
                    "type": "status"
                }
            ],
            "type": "prompt"
        }
    ],
    "console_title_template": "{{ .Folder }}",
    "final_space": true,
    "transient_prompt": {
        "background": "transparent",
        "foreground": "#d6deeb",
        "template": "\ue285 "
    },
    "version": 3
}


### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
Version: 29.9.0

Shell: shell (5.9)

Prompt:

╭─       test  0ms                                                                                                        default  "test" 3.12.13 16:52:48  22:52Z
╰─

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Root(false)                                -   4 ms
Path(true)                                 -   4 ms
Path(false)                                -   3 ms
Git(false)                                 -   0 ms
Executiontime(true)                        -   0 ms
Aws(true)                                  -   1 ms
Kubectl(false)                             -   0 ms
Node(false)                                -   0 ms
Python(true)                               -   7 ms
Rust(false)                                -   0 ms
Time(true)                                 -   0 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms
Status(true)                               -   0 ms

Run duration: 22.600041ms

Cache path: /Users/josephgruber/.cache/oh-my-posh

Config path: /Users/josephgruber/.dotfiles/themes/oh-my-posh.json

Logs:

[DEBUG] 16:52:48.453 debug.go:39 → logging enabled, raw mode: true
[DEBUG] 16:52:48.453 store.go:]:163 → (session) key not found: prompt_count_cache
[TRACE] 16:52:48.453 store.go:](session prompt_count_cache) - 192.416µs
[TRACE] 16:52:48.454 terminal.go:setPromptCount() - 197.792µs
[TRACE] 16:52:48.454 terminal.go:GOOS() - 42ns
[DEBUG] 16:52:48.454 terminal.go:setPwd:102 → /Users/josephgruber/Desktop/test
[TRACE] 16:52:48.454 terminal.go:setPwd() - 15.083µs
[TRACE] 16:52:48.454 terminal.go:Init() - 395.333µs
[DEBUG] 16:52:48.454 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.454 path.go:Path() - 182.458µs
[DEBUG] 16:52:48.454 store.go:init:95 → (device) loading is_wsl
[DEBUG] 16:52:48.454 store.go:init:95 → (device) loading DSC_CONFIGURATION
[DEBUG] 16:52:48.454 store.go:init:95 → (device) loading INITVERSIONZSH
[DEBUG] 16:52:48.454 store.go:init:95 → (device) loading DSC_SHELL
[TRACE] 16:52:48.454 store.go:init(device omp.cache) - 361.125µs
[TRACE] 16:52:48.454 init.go:SessionID() - 417ns
[TRACE] 16:52:48.454 path.go:Path() - 42ns
[DEBUG] 16:52:48.454 store.go:init:95 → (session) loading toggle_cache
[DEBUG] 16:52:48.454 store.go:init:95 → (session) loading CONFIG
[DEBUG] 16:52:48.454 store.go:init:95 → (session) loading prompt_count_cache
[DEBUG] 16:52:48.454 store.go:init:95 → (session) loading CONFIG_SOURCE
[DEBUG] 16:52:48.454 store.go:init:95 → (session) loading template_cache
[TRACE] 16:52:48.454 store.go:init(session zsh.c5e56627-ec3a-4e6c-84e1-e842d44fdd8e.omp.cache) - 126.375µs
[DEBUG] 16:52:48.454 store.go:]:163 → (device) key not found: reload
[TRACE] 16:52:48.454 store.go:](device reload) - 4.667µs
[DEBUG] 16:52:48.455 store.go:]:176 → (session) found entry: CONFIG - /gIHfwMBAQZDb25maWcB/4AAASABB1BhbGV0dGUB/4IAAQtEZWJ1Z1Byb21wdAH/hAABA1ZhcgH/jgABCFBhbGV0dGVzAf+QAAEJVmFsaWRMaW5lAf+EAAEPU2Vjb25kYXJ5UHJvbXB0Af+EAAEPVHJhbnNpZW50UHJvbXB0Af+EAAEJRXJyb3JMaW5lAf+EAAEETWFwcwH/lAABB1VwZ3JhZGUB/5gAAQdFeHRlbmRzAQwAAQtBY2NlbnRDb2xvcgEMAAEUQ29uc29sZVRpdGxlVGVtcGxhdGUBDAABA1BXRAEMAAEGU291cmNlAQwAAQZGb3JtYXQBDAABElRlcm1pbmFsQmFja2dyb3VuZAEMAAEOVG9vbFRpcHNBY3Rpb24BDAABBkJsb2NrcwH/ngABBUN5Y2xlAf+iAAENSVRlcm1GZWF0dXJlcwH/pAABCFRvb2x0aXBzAf+cAAEHVmVyc2lvbgEEAAENTWlncmF0ZUdseXBocwECAAEFQXN5bmMBAgABEFNoZWxsSW50ZWdyYXRpb24BAgABCkZpbmFsU3BhY2UBAgABDVVwZ3JhZGVOb3RpY2UBAgABDlBhdGNoUHdzaEJsZWVkAQIAAQtBdXRvVXBncmFkZQECAAEXRW5hYmxlQ3Vyc29yUG9zaXRpb25pbmcBAgABCVN0cmVhbWluZwEEAAAAF/+BBAEBB1BhbGV0dGUB/4IAAQwBDAAA/gIx/4MDAQEHU2VnbWVudAH/hAABJAEHT3B0aW9ucwH/hgABClByb3BlcnRpZXMB/4YAAQVDYWNoZQH/iAABBUFsaWFzAQwAAQ5MZWFkaW5nRGlhbW9uZAEMAAEPVHJhaWxpbmdEaWFtb25kAQwAAQhUZW1wbGF0ZQEMAAEKRm9yZWdyb3VuZAEMAAEOVGVtcGxhdGVzTG9naWMBDAABD1Bvd2VybGluZVN5bWJvbAEMAAEKQmFja2dyb3VuZAEMAAEGRmlsbGVyAQwAAQRUeXBlAQwAAQVTdHlsZQEMAAEWTGVhZGluZ1Bvd2VybGluZVN5bWJvbAEMAAELUGxhY2Vob2xkZXIBDAABBFRpcHMB/4oAARNCYWNrZ3JvdW5kVGVtcGxhdGVzAf+MAAEJVGVtcGxhdGVzAf+MAAEORXhjbHVkZUZvbGRlcnMB/4oAAQ5JbmNsdWRlRm9sZGVycwH/igABBU5lZWRzAf+KAAETRm9yZWdyb3VuZFRlbXBsYXRlcwH/jAABBUluZGV4AQQAAQhNaW5XaWR0aAEEAAEIRHVyYXRpb24BBAABCk5hbWVMZW5ndGgBBAABCE1heFdpZHRoAQQAAQdUaW1lb3V0AQQAAQdOZXdsaW5lAQIAAQdFbmFibGVkAQIAAQ9JbnZlcnRQb3dlcmxpbmUBAgABBUZvcmNlAQIAAQdUb2dnbGVkAQIAAQdQZW5kaW5nAQIAAQtJbnRlcmFjdGl2ZQECAAAAE/+FBAEBA01hcAH/hgABDAEQAAAt/4cDAQEFQ2FjaGUB/4gAAQIBCER1cmF0aW9uAQwAAQhTdHJhdGVneQEMAAAAFv+JAgEBCFtdc3RyaW5nAf+KAAEMAAAS/4sCAQEETGlzdAH/jAABDAAAJ/+NBAEBF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9Af+OAAEMARAAAC3/jwMBAQhQYWxldHRlcwH/kAABAgEETGlzdAH/kgABCFRlbXBsYXRlAQwAAAAp/5EEAQEYbWFwW3N0cmluZ11jb2xvci5QYWxldHRlAf+SAAEMAf+CAAA//5MDAQEGQ29uZmlnAf+UAAEDAQhVc2VyTmFtZQH/lgABCEhvc3ROYW1lAf+WAAEJU2hlbGxOYW1lAf+WAAAAE/+VBAEBA01hcAH/lgABDAEMAABc/5cDAQEGQ29uZmlnAf+YAAEGAQZTb3VyY2UBDAABCEludGVydmFsAQwAAQZMYXRlc3QBDAABBEF1dG8BAgABDURpc3BsYXlOb3RpY2UBAgABBUZvcmNlAQIAAAAe/50CAQEPW10qY29uZmlnLkJsb2NrAf+eAAH/mgAA/5D/mQMBAv+aAAEKAQRUeXBlAQwAAQlBbGlnbm1lbnQBDAABBkZpbGxlcgEMAAEIT3ZlcmZsb3cBDAABDkxlYWRpbmdEaWFtb25kAQwAAQ9UcmFpbGluZ0RpYW1vbmQBDAABCFNlZ21lbnRzAf+cAAEHTmV3bGluZQECAAEFRm9yY2UBAgABBUluZGV4AQQAAAAg/5sCAQERW10qY29uZmlnLlNlZ21lbnQB/5wAAf+EAAAU/6ECAQEFQ3ljbGUB/6IAAf+gAAAq/58DAQL/oAABAgEKQmFja2dyb3VuZAEMAAEKRm9yZWdyb3VuZAEMAAAAG/+jAgEBDUlUZXJtRmVhdHVyZXMB/6QAAQwAAP4KTP+ABwcE7oqFIAEHI2Q2ZGVlYgMLdHJhbnNwYXJlbnQAAwEDY2RuAQQxNjhoAAMNe3sgLkZvbGRlciB9fQI0L1VzZXJzL2pvc2VwaGdydWJlci8uZG90ZmlsZXMvdGhlbWVzL29oLW15LXBvc2guanNvbgEEanNvbgMDAQZwcm9tcHQBBGxlZnQFBgEBB3dpbmRvd3MGc3RyaW5nDAUAA+6YqgQJ4pWt4pSA7oK2Agsge3suSWNvbn19IAEHIzAxMTYyNwMHIzIxYzdhOAICb3MBB2RpYW1vbmQABwUg74qSIAEHI2ZmZWI5NQID7oKwAQcjZWY1MzUwAgRyb290AQlwb3dlcmxpbmUL/gFUAAEEC2ZvbGRlcl9pY29uBnN0cmluZwwGAATvgbwgFWZvbGRlcl9zZXBhcmF0b3JfaWNvbgZzdHJpbmcMEgAQPCMwMTE2Mjc+7oKxPC8+IAlob21lX2ljb24Gc3RyaW5nDAcABSDurIYgBXN0eWxlBnN0cmluZwwQAA5hZ25vc3Rlcl9zaG9ydAYbe3sgcGF0aCAuUGF0aCAuTG9jYXRpb24gfX0gAQcjMDExNjI3AgPugrABByM4MkFBRkYCBHBhdGgBCXBvd2VybGluZQv+AUAAAQQLZm9sZGVyX2ljb24Gc3RyaW5nDAYABO+BvCAVZm9sZGVyX3NlcGFyYXRvcl9pY29uBnN0cmluZwwSABA8IzAxMTYyNz7ugrE8Lz4gCWhvbWVfaWNvbgZzdHJpbmcMBwAFIO6shiAFc3R5bGUGc3RyaW5nDAgABmZvbGRlcgYbe3sgcGF0aCAuUGF0aCAuTG9jYXRpb24gfX0gAQcjMDExNjI3AgPugrABByM4MkFBRkYCBHBhdGgBCXBvd2VybGluZQ7+AT4AAQQLYnJhbmNoX2ljb24Gc3RyaW5nDAYABO6cpSAMZmV0Y2hfc3RhdHVzBGJvb2wCAgABE2ZldGNoX3Vwc3RyZWFtX2ljb24EYm9vbAICAAEPbWFwcGVkX2JyYW5jaGVzF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/44vAAIGZmVhdC8qBnN0cmluZwwHAAXwn5qAIAVidWcvKgZzdHJpbmcMBwAF8J+QmyAG/gFAIHt7IC5VcHN0cmVhbUljb24gfX0ge3sgLkhFQUQgfX17e2lmIC5CcmFuY2hTdGF0dXMgfX0ge3sgLkJyYW5jaFN0YXR1cyB9fXt7IGVuZCB9fXt7IGlmIC5Xb3JraW5nLkNoYW5nZWQgfX0g74GEIHt7IC5Xb3JraW5nLlN0cmluZyB9fXt7IGVuZCB9fXt7IGlmIGFuZCAoLldvcmtpbmcuQ2hhbmdlZCkgKC5TdGFnaW5nLkNoYW5nZWQpIH19IHx7eyBlbmQgfX17eyBpZiAuU3RhZ2luZy5DaGFuZ2VkIH19IO+BhiB7eyAuU3RhZ2luZy5TdHJpbmcgfX17eyBlbmQgfX17eyBpZiBndCAuU3Rhc2hDb3VudCAwIH19IO6tiyB7eyAuU3Rhc2hDb3VudCB9fXt7IGVuZCB9fSABByMwMTE2MjcCA+6CsAEHI2FkZGI2NwIDZ2l0AQlwb3dlcmxpbmUEBEF7eyBpZiBvciAoLldvcmtpbmcuQ2hhbmdlZCkgKC5TdGFnaW5nLkNoYW5nZWQpIH19I2U0Y2Y2YXt7IGVuZCB9fTl7eyBpZiBhbmQgKGd0IC5BaGVhZCAwKSAoZ3QgLkJlaGluZCAwKSB9fSNmNzhjNmN7eyBlbmQgfX0ke3sgaWYgZ3QgLkFoZWFkIDAgfX0jQzc5MkVBe3sgZW5kIH19JXt7IGlmIGd0IC5CZWhpbmQgMCB9fSNjNzkyZWF7eyBlbmQgfX0AAQIFc3R5bGUGc3RyaW5nDAsACXJvdW5kcm9jawl0aHJlc2hvbGQHZmxvYXQ2NAgCAAAFA+6CtAETIHt7IC5Gb3JtYXR0ZWRNcyB9fQEHI2Q2ZGVlYgMHIzU3NTY1NgINZXhlY3V0aW9udGltZQEHZGlhbW9uZAAAAQZwcm9tcHQBBXJpZ2h0AgVicmVhawMHBQPugrICOe6erSB7eyAuUHJvZmlsZSB9fXt7IGlmIC5SZWdpb24gfX1Ae3sgLlJlZ2lvbiB9fXt7IGVuZCB9fQEHI2ZhYTAyOQMHIzU2NTY1NgIDYXdzAQdkaWFtb25kC/4BLAABARBwYXJzZV9rdWJlY29uZmlnBGJvb2wCAgABBAPugrICOu6gnSB7ey5Db250ZXh0fX17e2lmIC5OYW1lc3BhY2V9fSA6OiB7ey5OYW1lc3BhY2V9fXt7ZW5kfX0BByNmZmZmZmYDByMzMTZjZTQCB2t1YmVjdGwBB2RpYW1vbmQL/gFAAAEDFWZldGNoX3BhY2thZ2VfbWFuYWdlcgRib29sAgIAAQhucG1faWNvbgZzdHJpbmcMEgAQPCNjYzNhM2E+7pyePC8+IAl5YXJuX2ljb24Gc3RyaW5nDBIAEDwjMzQ4Y2JhPu6apzwvPiAEA+6CsgJO7pyYIHt7IGlmIC5QYWNrYWdlTWFuYWdlckljb24gfX17eyAuUGFja2FnZU1hbmFnZXJJY29uIH19IHt7IGVuZCB9fXt7IC5GdWxsIH19AQcjM0M4NzNBAwcjMzAzMDMwAgRub2RlAQdkaWFtb25kC/4BLAAFA+6CsgJh7oi1ICB7eyBpZiAuRXJyb3IgfX17eyAuRXJyb3IgfX17eyBlbHNlIH19e3sgaWYgLlZlbnYgfX17eyAuVmVudiB9fSB7eyBlbmQgfX17eyAuRnVsbCB9fXt7IGVuZCB9fQEHI0ZGRTg3MwMHIzMwNjk5OAIGcHl0aG9uAQdkaWFtb25kC/4BLAAFA+6CsgI+7p6oIHt7IGlmIC5FcnJvciB9fXt7IC5FcnJvciB9fXt7IGVsc2UgfX17eyAuRnVsbCB9fXt7IGVuZCB9fSABByMwMDAwMDADByNmZmZmZmYCBHJ1c3QBB2RpYW1vbmQL/gEsAAEBC3RpbWVfZm9ybWF0BnN0cmluZwwKAAgxNTowNDowNQQD7oKyAibumYEge3sgLkN1cnJlbnREYXRlIHwgZGF0ZSAuRm9ybWF0IH19IAEHI2Q2ZGVlYgMHIzIzNGQ3MAIEdGltZQEHZGlhbW9uZAAFA+6CsgED7oK0ATDuvZ8ge3sgZGF0ZUluWm9uZSAiMTU6MDRaIiAuQ3VycmVudERhdGUgIlVUQyIgfX0BByNkNmRlZWIDByMyMzRkNzACBHRpbWUBB2RpYW1vbmQAAAEGcHJvbXB0AQRsZWZ0BQIHBuKVsOKUgAEHIzIxYzdhOAUEdGV4dAEFcGxhaW4AAQEOYWx3YXlzX2VuYWJsZWQEYm9vbAICAAEGBu6Khe6KhQEHIzIyZGE2ZQUGc3RhdHVzAQVwbGFpbgkBI3t7IGlmIGd0IC5Db2RlIDAgfX0jZWY1MzUwe3sgZW5kIH19AAEBAAQGBAEF/8gA
[TRACE] 16:52:48.455 store.go:](session CONFIG) - 387.792µs
[TRACE] 16:52:48.455 gob.go:Restore() - 89.416µs
[TRACE] 16:52:48.455 gob.go:Get() - 484.667µs
[TRACE] 16:52:48.455 terminal.go:Shell() - 42ns
[TRACE] 16:52:48.455 terminal.go:Flags() - 42ns
[DEBUG] 16:52:48.455 store.go:]:176 → (session) found entry: template_cache - {map[Aws:0x20f9bff2b90 Executiontime:0x20f9c057ec0 Os:0x20f9c057cc0 Path:0x20f9be62540 Python:0x20f9bfc2700 Status:0x20f9bff2d20 Text:0x20f9c0700c0 Time:0x20f9bff2c30] map[] ~/Desktop/test test  josephgruber macbook-pro-wg.internal 5.9 zsh /Users/josephgruber/Desktop/test darwin 29.9.0 47 2 0 0 false false}
[TRACE] 16:52:48.455 store.go:](session template_cache) - 25.375µs
[TRACE] 16:52:48.455 cache.go:restoreCache() - 30µs
[DEBUG] 16:52:48.455 writer.go:Init:107 → terminal program: ghostty
[DEBUG] 16:52:48.455 writer.go:Init:108 → terminal shell: shell
[DEBUG] 16:52:48.455 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.455 terminal.go:Getenv(OMP_CACHE_DISABLED) - 2.375µs
[DEBUG] 16:52:48.455 store.go:]:163 → (device) key not found: accent_color
[TRACE] 16:52:48.455 store.go:](device accent_color) - 3.709µs
[ERROR] 16:52:48.463 terminal.go:RunCommand:277 → exit status 1
[DEBUG] 16:52:48.463 terminal.go:RunCommand:280 ↓
    2026-03-21 16:52:48.462 defaults[95761:14920436]
    The domain/default pair of (kCFPreferencesAnyApplication, AppleAccentColor) does not exist
[TRACE] 16:52:48.463 terminal.go:RunCommand(defaults read -g AppleAccentColor) - 8.062375ms
[ERROR] 16:52:48.463 colors_darwin.go:GetAccentColor:14 → exit status 1
[TRACE] 16:52:48.463 colors.go:SetAccentColor() - 8.074583ms
[TRACE] 16:52:48.463 terminal.go:Shell() - 125ns
[DEBUG] 16:52:48.463 terminal.go:Getenv:64 → 5.9
[TRACE] 16:52:48.463 terminal.go:Getenv(POSH_SHELL_VERSION) - 7.708µs
[DEBUG] 16:52:48.463 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 16:52:48.463 text.go:patchTemplate:149 → {{ .Folder }}
[TRACE] 16:52:48.463 terminal.go:Flags() - 41ns
[DEBUG] 16:52:48.463 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.463 terminal.go:Getenv(POSH_CURSOR_LINE) - 2.083µs
[DEBUG] 16:52:48.463 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.463 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 1.25µs
[TRACE] 16:52:48.463 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 0s
[TRACE] 16:52:48.463 terminal.go:Flags() - 0s
[TRACE] 16:52:48.463 terminal.go:Flags() - 41ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 41ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 0s
[TRACE] 16:52:48.463 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 41ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 83ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 83ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.463 terminal.go:Flags() - 83ns
[DEBUG] 16:52:48.463 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.463 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.463 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.463 terminal.go:GOOS() - 42ns
[TRACE] 16:52:48.463 terminal.go:GOOS() - 83ns
[DEBUG] 16:52:48.463 segment.go:Execute:173 → segment: Executiontime
[TRACE] 16:52:48.463 terminal.go:GOOS() - 42ns
[DEBUG] 16:52:48.463 segment.go:Execute:173 → segment: Path
[DEBUG] 16:52:48.463 segment.go:Execute:173 → segment: Git
[DEBUG] 16:52:48.463 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.463 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.463 store.go:](session toggle_cache) - 4.583µs
[DEBUG] 16:52:48.463 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.463 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.463 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.463 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.463 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 16:52:48.463 store.go:](session toggle_cache) - 8.666µs
[TRACE] 16:52:48.463 terminal.go:GOOS() - 42ns
[TRACE] 16:52:48.463 terminal.go:GOOS() - 250ns
[DEBUG] 16:52:48.463 segment.go:isToggled:291 → no toggles found
[TRACE] 16:52:48.463 store.go:](session toggle_cache) - 6.916µs
[TRACE] 16:52:48.463 store.go:](session toggle_cache) - 6.709µs
[DEBUG] 16:52:48.463 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.463 segment.go:Execute:173 → segment: Os
[DEBUG] 16:52:48.463 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.463 map.go:Bool:133 → always_enabled: false
[TRACE] 16:52:48.463 terminal.go:ExecutionTime() - 42ns
[DEBUG] 16:52:48.463 map.go:Float64:155 → threshold: 0.000000
[DEBUG] 16:52:48.463 map.go:Bool:133 → disable_with_jj: false
[DEBUG] 16:52:48.463 map.go:String:83 → style: roundrock
[DEBUG] 16:52:48.463 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 16:52:48.463 store.go:](session toggle_cache) - 11.5µs
[TRACE] 16:52:48.463 terminal.go:Flags() - 0s
[TRACE] 16:52:48.463 store.go:](session toggle_cache) - 11.375µs
[DEBUG] 16:52:48.463 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.463 segment.go:isToggled:291 → no toggles found
[TRACE] 16:52:48.463 terminal.go:GOOS() - 0s
[DEBUG] 16:52:48.463 map.go:String:79 → macos: 
[DEBUG] 16:52:48.463 terminal.go:ResolveSymlink:195 → /Users/josephgruber/Desktop/test
[TRACE] 16:52:48.463 terminal.go:Flags() - 41ns
[TRACE] 16:52:48.463 terminal.go:ResolveSymlink(/Users/josephgruber/Desktop/test) - 25.75µs
[ERROR] 16:52:48.463 terminal.go:HasParentFilePath:464 → stat .git: no such file or directory
[TRACE] 16:52:48.463 terminal.go:HasParentFilePath(.git) - 77.125µs
[TRACE] 16:52:48.463 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.463 text.go:patchTemplate:149 →  {{.Data.Icon}}
[DEBUG] 16:52:48.467 terminal_unix.go:TerminalWidth:83 → terminal width: 180
[TRACE] 16:52:48.467 terminal_unix.go:TerminalWidth() - 3.5655ms
[TRACE] 16:52:48.467 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.467 terminal_unix.go:TerminalWidth:83 → terminal width: 180
[TRACE] 16:52:48.467 terminal_unix.go:TerminalWidth() - 3.744958ms
[DEBUG] 16:52:48.467 terminal_unix.go:TerminalWidth:83 → terminal width: 180
[TRACE] 16:52:48.467 terminal_unix.go:TerminalWidth() - 3.75125ms
[DEBUG] 16:52:48.467 map.go:Bool:133 → display_cygpath: false
[TRACE] 16:52:48.467 terminal_unix.go:Root() - 334ns
[TRACE] 16:52:48.467 terminal.go:GOOS() - 0s
[TRACE] 16:52:48.467 separator.go:Separator() - 42ns
[TRACE] 16:52:48.467 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.467 terminal.go:Shell() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 41ns
[DEBUG] 16:52:48.467 map.go:Bool:133 → mapped_locations_enabled: true
[DEBUG] 16:52:48.467 map.go:String:79 → windows_registry_icon: 
[DEBUG] 16:52:48.467 map.go:String:79 → windows_registry_icon: 
[DEBUG] 16:52:48.467 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.467 separator.go:Separator() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 41ns
[TRACE] 16:52:48.467 terminal.go:GOOS() - 0s
[DEBUG] 16:52:48.467 map.go:String:83 → home_icon:  
[DEBUG] 16:52:48.467 map.go:KeyValueMap:208 → mapped_locations: map[]
[TRACE] 16:52:48.467 separator.go:Separator() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 0s
[TRACE] 16:52:48.467 separator.go:Separator() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 42ns
[TRACE] 16:52:48.467 terminal.go:GOOS() - 42ns
[TRACE] 16:52:48.467 terminal.go:GOOS() - 0s
[TRACE] 16:52:48.467 terminal.go:GOOS() - 42ns
[DEBUG] 16:52:48.467 map.go:String:79 → gitdir_format:
[DEBUG] 16:52:48.467 map.go:String:83 → style: agnoster_short
[DEBUG] 16:52:48.467 map.go:Bool:133 → display_root: false
[DEBUG] 16:52:48.467 map.go:Int:178 → max_depth: 1
[DEBUG] 16:52:48.467 map.go:Bool:133 → hide_root_location: false
[DEBUG] 16:52:48.467 map.go:String:83 → folder_icon: 
[DEBUG] 16:52:48.467 map.go:StringArray:220 → cycle: []
[DEBUG] 16:52:48.467 map.go:String:79 → folder_separator_template:
[DEBUG] 16:52:48.467 map.go:String:83 → folder_separator_icon: <#011627></>
[DEBUG] 16:52:48.467 map.go:Bool:133 → cycle_folder_separator: false
[DEBUG] 16:52:48.467 map.go:String:79 → folder_format: %s
[DEBUG] 16:52:48.467 map.go:String:79 → edge_format: %s
[DEBUG] 16:52:48.467 map.go:String:79 → left_format: %s
[DEBUG] 16:52:48.467 map.go:String:79 → right_format: %s
[TRACE] 16:52:48.467 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.467 terminal.go:GOOS() - 41ns
[TRACE] 16:52:48.467 terminal.go:StackCount() - 41ns
[TRACE] 16:52:48.467 terminal_unix.go:DirIsWritable(/Users/josephgruber/Desktop/test) - 13.5µs
[TRACE] 16:52:48.467 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.467 text.go:patchTemplate:149 → {{ path .Data.Path .Data.Location }}
[DEBUG] 16:52:48.468 text.go:patchTemplate:149 →  {{ .Data.FormattedMs }}
[TRACE] 16:52:48.468 terminal.go:Shell() - 0s
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[TRACE] 16:52:48.468 terminal.go:Flags() - 41ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[TRACE] 16:52:48.468 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[TRACE] 16:52:48.468 terminal.go:Flags() - 41ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 41ns
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.468 terminal.go:GOOS() - 0s
[TRACE] 16:52:48.468 terminal.go:GOOS() - 0s
[DEBUG] 16:52:48.468 segment.go:Execute:173 → segment: Aws
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.468 terminal.go:GOOS() - 41ns
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.468 segment.go:Execute:173 → segment: Node
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 3.875µs
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 7.209µs
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.468 terminal_unix.go:TerminalWidth:63 → terminal width: 180
[DEBUG] 16:52:48.468 map.go:String:79 → time_format: 15:04:05
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 5.875µs
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.468 terminal_unix.go:TerminalWidth:63 → terminal width: 180
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.468 terminal_unix.go:TerminalWidth() - 2.541µs
[TRACE] 16:52:48.468 terminal.go:GOOS() - 0s
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 segment.go:Execute:173 → segment: Time
[TRACE] 16:52:48.468 terminal.go:GOOS() - 41ns
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 2.25µs
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.468 map.go:String:83 → time_format: 15:04:05
[DEBUG] 16:52:48.468 segment.go:Execute:173 → segment: Kubectl
[TRACE] 16:52:48.468 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 84ns
[TRACE] 16:52:48.468 terminal.go:Flags() - 166ns
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 9.959µs
[DEBUG] 16:52:48.468 map.go:StringArray:220 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 map.go:StringArray:220 → folders: []
[DEBUG] 16:52:48.468 terminal_unix.go:TerminalWidth:63 → terminal width: 180
[TRACE] 16:52:48.468 terminal.go:GOOS() - 42ns
[DEBUG] 16:52:48.468 map.go:Bool:133 → home_enabled: false
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.468 terminal_unix.go:TerminalWidth() - 2.125µs
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 segment.go:Execute:173 → segment: Rust
[TRACE] 16:52:48.468 terminal.go:GOOS() - 41ns
[DEBUG] 16:52:48.468 map.go:String:79 → display_mode: files
[DEBUG] 16:52:48.468 map.go:Bool:141 → parse_kubeconfig: true
[DEBUG] 16:52:48.468 segment.go:Execute:173 → segment: Python
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.468 terminal.go:Getenv(KUBECONFIG) - 2.042µs
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 3.625µs
[DEBUG] 16:52:48.468 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[TRACE] 16:52:48.468 store.go:](session toggle_cache) - 9.458µs
[DEBUG] 16:52:48.468 terminal_unix.go:TerminalWidth:63 → terminal width: 180
[DEBUG] 16:52:48.468 segment.go:isToggled:291 → no toggles found
[TRACE] 16:52:48.468 terminal_unix.go:TerminalWidth() - 2.166µs
[DEBUG] 16:52:48.468 terminal_unix.go:TerminalWidth:63 → terminal width: 180
[TRACE] 16:52:48.468 terminal_unix.go:TerminalWidth() - 1.959µs
[DEBUG] 16:52:48.468 map.go:String:79 → display_mode: environment
[DEBUG] 16:52:48.468 map.go:StringArray:220 → extensions: [*.rs Cargo.toml Cargo.lock]
[DEBUG] 16:52:48.468 map.go:StringArray:220 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 16:52:48.468 map.go:StringArray:220 → folders: []
[DEBUG] 16:52:48.468 map.go:StringArray:220 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 16:52:48.468 map.go:Bool:133 → home_enabled: false
[DEBUG] 16:52:48.468 map.go:Bool:133 → home_enabled: false
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 map.go:String:79 → display_mode: files
[DEBUG] 16:52:48.468 map.go:Bool:133 → fetch_virtual_env: true
[DEBUG] 16:52:48.468 terminal.go:FileContent:212 ↓
    apiVersion: v1
    kind: Config
    clusters: []
    users: []
    contexts: []
    preferences: {}
    current-context: ''

[TRACE] 16:52:48.468 terminal.go:FileContent(/Users/josephgruber/.kube/config) - 48.75µs
[DEBUG] 16:52:48.468 terminal.go:CommandPath:304 → /Users/josephgruber/Desktop/test/.venv/bin/python
[TRACE] 16:52:48.468 terminal.go:CommandPath(python) - 38.958µs
[DEBUG] 16:52:48.468 map.go:Bool:133 → display_error: false
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(pyvenv.cfg) - 55µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:146 → true
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(pyvenv.cfg) - 32.959µs
[DEBUG] 16:52:48.468 map.go:Bool:133 → display_default: true
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.468 terminal.go:Getenv(AWS_VAULT) - 2.791µs
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.468 terminal.go:Getenv(AWS_DEFAULT_PROFILE) - 1.208µs
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → default
[TRACE] 16:52:48.468 terminal.go:Getenv(AWS_PROFILE) - 1.417µs
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.468 terminal.go:Getenv(AWS_REGION) - 1.458µs
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.468 terminal.go:Getenv(AWS_DEFAULT_REGION) - 1.125µs
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → NO DATA
[TRACE] 16:52:48.468 terminal.go:Getenv(AWS_CONFIG_FILE) - 1.5µs
[DEBUG] 16:52:48.468 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.468 terminal.go:FileContent:212 ↓
    home = /Users/josephgruber/.pyenv/versions/3.12.13/bin
    implementation = CPython
    version_info = 3.12.13.final.0
    version = 3.12.13
    executable = /Users/josephgruber/.pyenv/versions/3.12.13/bin/python3.12
    command = /opt/homebrew/Cellar/pipenv/2026.2.1/libexec/bin/python -m virtualenv /Users/josephgruber/Desktop/test/.venv
    virtualenv = 21.2.0
    prompt = "test"
    include-system-site-packages = false
    base-prefix = /Users/josephgruber/.pyenv/versions/3.12.13
    base-exec-prefix = /Users/josephgruber/.pyenv/versions/3.12.13
    base-executable = /Users/josephgruber/.pyenv/versions/3.12.13/bin/python

[TRACE] 16:52:48.468 terminal.go:FileContent(/Users/josephgruber/Desktop/test/.venv/pyvenv.cfg) - 37.666µs
[DEBUG] 16:52:48.468 map.go:StringArray:220 → tooling: [pyenv python python3 py]
[DEBUG] 16:52:48.468 map.go:Bool:133 → fetch_version: true
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(pnpm-lock.yaml) - 263.584µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(yarn.lock) - 2.916µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(bun.lockb) - 1.791µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(bun.lock) - 2.083µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(package-lock.json) - 2µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(package.json) - 1.958µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(*.rs) - 442.792µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(*.js) - 191.084µs
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(Cargo.toml) - 4µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(*.ts) - 4.583µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(Cargo.lock) - 5.75µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(package.json) - 6.375µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(.nvmrc) - 4.458µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(pnpm-workspace.yaml) - 2.959µs
[DEBUG] 16:52:48.468 map.go:StringArray:220 → tooling: [rustc]
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:Flags() - 0s
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(.pnpmfile.cjs) - 2.625µs
[DEBUG] 16:52:48.468 terminal.go:HasFilesInDir:151 → false
[TRACE] 16:52:48.468 terminal.go:HasFilesInDir(.vue) - 2.291µs
[DEBUG] 16:52:48.468 map.go:StringArray:220 → tooling: [node]
[TRACE] 16:52:48.468 terminal.go:Flags() - 42ns
[DEBUG] 16:52:48.468 store.go:]:163 → (device) key not found: version_python
[TRACE] 16:52:48.468 store.go:](device version_python) - 448.208µs
[DEBUG] 16:52:48.468 terminal.go:CommandPath:297 → /Users/josephgruber/Desktop/test/.venv/bin/python
[TRACE] 16:52:48.468 terminal.go:CommandPath(python) - 25.125µs
[DEBUG] 16:52:48.468 terminal.go:Getenv:64 → /Users/josephgruber/.pyenv
[TRACE] 16:52:48.468 terminal.go:Getenv(PYENV_ROOT) - 2µs
[ERROR] 16:52:48.468 language.go:setVersion:238 → executable at /Users/josephgruber/Desktop/test/.venv/bin/python is not a pyenv shim
[DEBUG] 16:52:48.468 terminal.go:CommandPath:297 → /Users/josephgruber/Desktop/test/.venv/bin/python
[TRACE] 16:52:48.468 terminal.go:CommandPath(python) - 3.5µs
[TRACE] 16:52:48.468 terminal.go:HasCommand(python) - 5.458µs
[DEBUG] 16:52:48.468 terminal.go:FileContent:212 ↓
  REDACTED
[TRACE] 16:52:48.469 terminal.go:FileContent(/Users/josephgruber/.aws/config) - 714.958µs
[TRACE] 16:52:48.469 terminal.go:Flags() - 42ns
[DEBUG] 16:52:48.469 text.go:patchTemplate:149 →  {{ .Data.Profile }}{{ if .Data.Region }}@{{ .Data.Region }}{{ end }}
[DEBUG] 16:52:48.475 terminal.go:RunCommand:280 → Python 3.12.13
[TRACE] 16:52:48.475 terminal.go:RunCommand(python --version) - 6.433792ms
[DEBUG] 16:52:48.475 map.go:String:79 → version_url_template: https://docs.python.org/release/{{ .Major }}.{{ .Minor }}.{{ .Patch }}/whatsnew/changelog.html#python-{{ .Major }}-{{ .Minor }}-{{ .Patch }}
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 → https://docs.python.org/release/{{ .Data.Major }}.{{ .Data.Minor }}.{{ .Data.Patch }}/whatsnew/changelog.html#python-{{ .Data.Major }}-{{ .Data.Minor }}-{{ .Data.Patch }}
[DEBUG] 16:52:48.475 map.go:String:79 → cache_duration: none
[TRACE] 16:52:48.475 store.go:](device version_python) - 334ns
[TRACE] 16:52:48.475 terminal.go:Flags() - 750ns
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 →   {{ if .Data.Error }}{{ .Data.Error }}{{ else }}{{ if .Data.Venv }}{{ .Data.Venv }} {{ end }}{{ .Data.Full }}{{ end }}
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 →  {{ .Data.CurrentDate | date .Data.Format }}
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 →  {{ dateInZone "15:04Z" .Data.CurrentDate "UTC" }}
[DEBUG] 16:52:48.475 terminal_unix.go:TerminalWidth:63 → terminal width: 180
[TRACE] 16:52:48.475 terminal_unix.go:TerminalWidth() - 3.209µs
[DEBUG] 16:52:48.475 engine.go:shouldFill:165 → no filler specified
[TRACE] 16:52:48.475 terminal.go:Shell() - 42ns
[TRACE] 16:52:48.475 terminal.go:Flags() - 0s
[TRACE] 16:52:48.475 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.475 terminal.go:Flags() - 0s
[TRACE] 16:52:48.475 terminal.go:Flags() - 83ns
[DEBUG] 16:52:48.475 home.go:12 → /Users/josephgruber
[DEBUG] 16:52:48.475 home.go:12 → /Users/josephgruber
[TRACE] 16:52:48.475 terminal.go:GOOS() - 84ns
[TRACE] 16:52:48.475 terminal.go:GOOS() - 41ns
[DEBUG] 16:52:48.475 segment.go:Execute:173 → segment: Status
[DEBUG] 16:52:48.475 segment.go:Execute:173 → segment: Text
[DEBUG] 16:52:48.475 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 16:52:48.475 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 16:52:48.475 store.go:](session toggle_cache) - 4.125µs
[TRACE] 16:52:48.475 store.go:](session toggle_cache) - 3.584µs
[DEBUG] 16:52:48.475 segment.go:isToggled:291 → no toggles found
[DEBUG] 16:52:48.475 segment.go:isToggled:291 → no toggles found
[TRACE] 16:52:48.475 terminal.go:StatusCodes() - 125ns
[TRACE] 16:52:48.475 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.475 map.go:String:79 → status_template: {{ .Code }}
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 → {{ .Code }}
[DEBUG] 16:52:48.475 map.go:Bool:141 → always_enabled: true
[TRACE] 16:52:48.475 terminal.go:Flags() - 0s
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 → {{ if gt .Code 0 }}#ef5350{{ end }}
[DEBUG] 16:52:48.475 text.go:patchTemplate:149 → {{ if gt .Code 0 }}#ef5350{{ end }}
[TRACE] 16:52:48.475 terminal.go:Shell() - 42ns
[TRACE] 16:52:48.475 terminal.go:Flags() - 42ns
[TRACE] 16:52:48.475 terminal.go:Shell() - 42ns
[TRACE] 16:52:48.476 path.go:Path() - 42ns
```

--- a/src/segments/python.go
+++ b/src/segments/python.go
@@ -214,7 +214,7 @@ func (p *Python) pyvenvCfgPrompt() string {
 		key = strings.TrimSpace(key)
 		if key == "prompt" {
 			value := strings.TrimSpace(value)
-			return value
+			return strings.Trim(value, "\"")
 		}
 	}
 

--- a/src/segments/python_test.go
+++ b/src/segments/python_test.go
@@ -89,6 +89,15 @@ func TestPythonTemplate(t *testing.T) {
 			PyvenvCfg:      "home = /usr/bin/\nprompt = pyvenvCfgPrompt\n",
 			Expected:       "pyvenvCfgPrompt 3.8",
 		},
+		{
+			Case:           "pyvenv.cfg prompt with quotes (pipenv)",
+			FetchVersion:   true,
+			VirtualEnvName: "VENV",
+			PythonPath:     "/home/user/.pyenv/shims/python",
+			Template:       "{{ if .Venv }}{{ .Venv }} {{ end }}{{ .Major }}.{{ .Minor }}",
+			PyvenvCfg:      "home = /usr/bin/\nprompt = \"myproject\"\n",
+			Expected:       "myproject 3.8",
+		},
 	}
 
 	for _, tc := range cases {

